src/js/components/form-elements/TableBodyVirtualized.jsx

```diff
@@ -59,31 +61,31 @@ class TableBodyVirtualized extends Component {
         }
       });
     }
-
     return height || 28;
   }
 
   getRowHeight({ index }) {
     const { fieldsConfig: { subfieldKey, getDynamicRowAttr }, fields, properties } = this.props;
+    const rowValues = fields.value ? fields.value[index] : null;
```

Comment:
Can't you use null safe operator here?

Reply:
sure I could, but I was just copying the code from bottom to the top, so didn't even much care what was there in details. 
But it also depends if the `rowValues` could be `undefined` in this case, because if I used null safe operator, it would evaluate to `undefined`, so I gotta check if there is none strict checking if `rowValues` is `=== null`.